### ISSUE TITLE
Batchnorm and dropout?

### ISSUE THREAD

**Issue by za13** [qubvel/segmentation_models#189]
For the code in the "multiclass segmentation (camvid).ipynb" in "examples", the Unet uses batch normalization but does not use Dropout layers, right?

**Comment by qubvel** [comment]
yes
